public/js/test/mochitest/head.js

```diff
@@ -204,3 +221,27 @@ function isVisibleWithin(outerEl, innerEl) {
   return innerRect.top > outerRect.top &&
     innerRect.bottom < outerRect.bottom;
 }
+
+
+const selectors = {
+  callStackArrow: ".call-stack-pane .arrow",
+  frame: (index) => `.frames ul li:nth-child(${index})`
+}
```

Comment:
Yep, that was my concern too, that we'd be replicating selectors and it'd get confusing... this pattern is great though. I don't think we should be too stringent about keeping selectors out of tests, but common ones should certainly be in here.